react/components/Helpers.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export const Spacer = ({vertical=0, horizontal=0, index=1}) => (
+    <div
+      className={`bg-base w-100 z-${index} relative`}
+      style={{
+        height: vertical,
+        width: horizontal,
+      }}
+    />
+)
+  
+export const Border = ({fixed, top}) => (
+    <div
+      className={`${fixed && 'fixed top-0 left-0 w-100'} bb bw1 b--muted-4`}
```

Comment:
Use `classNames` instead

The `&&` operator will return the last truthy operand or the first operand, whether it is truth or not. If `fixed` receives `null` or `undefined`, the returned string for the classname will be
```"null bb bw1 b--muted-4"```ou ``` "undefined bb bw1 b--muted-4"```